event/register_oracle_event.go

```diff
@@ -22,8 +38,19 @@ func (e RegisterOracleEvent) GetEventAttributeValue() string {
 }
 
 func (e RegisterOracleEvent) EventHandler(event ctypes.ResultEvent) error {
-	// TODO: Verifying Remote Attestation and Executing Vote Txs
+	addressValue := event.Events[types.EventTypeRegistrationVote+"."+types.AttributeKeyOracleAddress][0]
+
+	uniqueID := hex.EncodeToString(e.reactor.EnclaveInfo().UniqueID)
+	oracleRegistration, err := e.reactor.GRPCClient().GetOracleRegistration(addressValue, uniqueID)
+	if err != nil {
+		return err
+	}
+
+	err = sgx.VerifyRemoteReport(oracleRegistration.NodePubKeyRemoteReport, oracleRegistration.NodePubKey, *e.reactor.EnclaveInfo())
```

Comment:
When I tested it, it fails here.
```
time="2022-09-07T02:41:08Z" level=error msg="failed to handle event 'message.action='RegisterOracle'': invalid data in the report"
```

We hashed the NodePubKey to SHA256 and put int the remoteReport.
https://github.com/medibloc/panacea-doracle/blob/main/cmd/doracled/cmd/register_oracle.go#L167

Reply:
Fixed in [4e92461](https://github.com/medibloc/panacea-doracle/pull/32/commits/4e92461808d4f2de1e9d67ce096d89cf9fb3a1c6).